feature(readme-branch): update readme

carolinaberrafato opened the pull request:
## What I did

- Branch update to introduce the client, team and project organization
- Tecnologies used to build the project and how we'll run it
   - I put in the project how we organize its stages
   - It also includes commit and branch patterns

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout "feature/readme-branch"
   1. git pull origin "feature/readme-branch"
   1. cd ./server
   1. yarn install
   1. node index.js

1. In VScode open "README.md"

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
*

Resolves:** "Atualizar README.md"

carolinaberrafato committed:
feature(readme-branch): update readme

MauricioFGF commented on README.md:
```diff
@@ -1,57 +1,53 @@
+- MongoDB
+- Keystone
+- Node Js
 
```
add Reactjs

Shellyda commented on README.md:
```diff
@@ -1,57 +1,53 @@
+O projeto foi dividido em algumas etapas. São elas:
 
-## 4. Create a env file
-Create a file named `.env` and, inside of it, place this:
+1. Produção do *wireframe*, seguida por sua validação;
+2. Produção do *mockup de alta fidelidade*, seguida por sua validação;
+3. *Sprint 1 de desenvolvimento*, onde será feita a estruturação do back-end;
+4. *Sprint 2 de desnvolvimento*, onde será feita a componentização do front-end;
+5. *Sprint 3 de desenvolvimento*, cujos objetivos são integração e deploy;
+6. *Entrega final do projeto*;
 
-    PORT=[PORT]
-    MONGO_URI=[MONGO_URI]
-    COOKIE_SECRET=[COOKIE_SECRET]
-    CLOUDINARY_URL=[CLOUDINARY_URL]
+As atividades são divididas entre os membres, que se encontram diariamente para alinhamento no projeto. Cada issue possui prazo e é marcada por tags específicas (por ordem de prioridade por exemplo), e a organização é pelo Notion e GitHub! 
```
Hi Carol! Congratulations on your work, you're awesome! <3 

Just some details, this Readme should be written in English, sorry if didn't make it clear, and in the part "How are we going to run the project?" referred to how to start the project technically, like Edgar's description here:

**Running in development**
To run this project in development mode, we will need to run two servers, the react one on /client and keystone on /server.

The command to run react is yarn start or npm start depending on which tool was used on installation, the react server will run on port 3000 by default.

Before running the keystone server, go to /server/updates/0.0.1-admin.js and change the admin user as you want, this user will be the first created, but you will be able to create others and delete this one later.

To run keystoneJS server, use the command node index.js, the server will run on whatever port is in the variale in the env file, you will find the admin interface in http://localhost:[PORT]/admin

**Running in production**
To run the server in production, go to /client and run the command yarn server, this command will create a react production optimized build and move it to /server.

Then go to /server and run node index.js, you will find the project on http://localhost:[PORT]

-----------------------
It's really cool that you brought the information about the sprints, I hadn't thought of that way! Congratulations!

No need to undo this part, just add the technical description as well. Thanks :)

carolinaberrafato committed:
bugfix(readme-branch): translation and small updates

carolinaberrafato committed:
bugfix(readme-branch): readme translation and small text change update

carolinaberrafato commented:
## README translation update

- Changed the language of README to English 
- Added "React.js" in "technologies used"

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout feature/readme-branch
   1. git pull origin feature/readme-branch
   1. cd ./server
   1. yarn install
   1. node index.js

1. In VScode open README.md

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
*
Time spent: <see your time spent on https://wakatime.com/dashboard >

Resolves:** README language and technologies used
